dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+# Copyright (c) 2015-2022, RTE (http://www.rte-france.com)
```

Comment:
No modifs of existing copyright

Reply:
One of these days, I will refrain my urges!